### ISSUE TITLE
Admin-facing "quick grade" does not work for group assignments (NOTE: This is not actually a bug)

### ISSUE THREAD

**Issue by maskib** [UM-USElab/gradecraft-development#4336]
### Expected Behavior
I would expect to be able to use the "quick grade" option for group assignments the same way I use it for individual assignments, if the option exists.

### Actual Behavior
The "quick grade" option still exists for group assignments. However, if you click it, you get an empty table with the "submit grades" and "cancel" buttons at the bottom. 

<img width="1552" alt="Screen Shot 2019-07-15 at 2 54 22 PM" src="https://user-images.githubusercontent.com/38505166/61241097-d1a3d080-a710-11e9-821d-565ece8e5087.png">

### Steps to reproduce
1. Create a group assignment. 
2. Click the "options" gear icon and select "quick grade" from the dropdown.

**Comment by adishy** [comment]
@maskib 
* After some quick testing I believe this issue is not because the "quick grade" page does not work but because there were no groups added for this particular assignment. In the local environment, I could access the "quick grade" page for all the assignments as they had groups added to them. Currently, I added a group for the assignment mentioned in the issue and the quick grade page now lists the group in the "quick grade" page.
* This can be made clear by added an empty state for group assignments to clarify to instructors that assignments for which no groups have been added cannot be graded without adding groups.
